src/core.js

```diff
@@ -330,6 +367,7 @@ async function pruneNonCriticalCssLauncher ({
     // -> [BLOCK FOR] critical css selector pruning (in browser)
     let criticalSelectors
     try {
+      await page.waitFor(300) // This is needed to  fix the Target closed error of puppeteer
```

Comment:
I would still prefer for this to be a prop. You asked who cares, and as I've mentioned before I run tens of thousands of penthouse jobs per day, so I do care (about this, and penthouse execution timings in general). :)

Reply:
My error case wasn't reported just by me. I read it 2 or 3 times in your issues list. So I guess you shouldn't ignore it. 
I can do it with an additional property. That is not a problem, but I would prefer a stable software which I can rely on. It doesn't matter if code runs 300ms faster per execution when there is a chance of getting errored results. 
But if you wish I will create a prop for that case.